content/sensu-go/5.21/operations/control-access/oidc-auth.md

```diff
@@ -429,6 +429,12 @@ For example, if you have an Okta group `groups` and you set the `groups_prefix`
 3. Add the `redirect_uri` configuration attribute in the [OIDC scope][25] and set the value to the Redirect URI configured at step 3 of [Create an Okta application][50]:
    - `"redirect_uri": "API_URL/api/enterprise/authentication/v2/oidc/callback"`
 
+4. Configure [authorization][3] via for your OIDC users and groups by creating [roles (or cluster roles)][4] and [role bindings (or cluster role bindings)][13] that map to the user and group names.
```

Comment:
superflous "via" here?

Reply:
The same comment applies to all the versions.

Reply:
Yes, thank you!